.github/CODEOWNERS

```diff
@@ -65,9 +65,12 @@
 /airflow/providers/google/ @turbaszek
 /airflow/providers/snowflake/ @turbaszek @potiuk @mik-laj
 /airflow/providers/cncf/kubernetes @jedcunningham
+/docs/apache-airflow-providers-cncf-kubernetes @jedcunningham
```

Comment:
I think this is supposed to be the last line of this section

Reply:
I was wondering if we have a way to catch docs, code and tests together rather than seperated line for each

Reply:
I think not. Look here https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
Glob (used there) is rather limited - we would have to rename the docs to have the same structure as providers to get a 'sane' way of specifying it (which might be a good idea btw before we split providers).

Reply:
:(